stdlib/InteractiveUtils/test/runtests.jl

```diff
@@ -173,7 +173,7 @@ module Tmp14173
 end
 varinfo(Tmp14173) # warm up
 const MEMDEBUG = ccall(:jl_is_memdebug, Bool, ())
-@test @allocated(varinfo(Tmp14173)) < (MEMDEBUG ? 60000 : 20000)
+@test @allocated(varinfo(Tmp14173)) < (MEMDEBUG ? 180000 : 60000)
```

Comment:
Related?

Reply:
Yeah, we're specifically measuring the amount of effort required to print the `Matrix` type, which went up due to this PR